packages/augur-ui/src/services/augursdk.ts

```diff
@@ -50,7 +50,7 @@ export class SDK {
       account
     );
 
-    const connector = (new WebWorkerConnector());
+    const connector = (iOSSafari ? new SEOConnector(): new WebWorkerConnector());
```

Comment:
do all the eventing/emitting work on SEOConnector?

Reply:
Yes. See attached.
![image](https://user-images.githubusercontent.com/328965/66780847-b159ee00-ee86-11e9-8262-cc78bccf6c28.png)